Docker-compose-web-srv-https-redirect/SyncFldr/centos-nginx/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos:8.4.2105
+
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*; \
+ sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*; \
+ dnf install -y epel-release nginx; \
+ mkdir -p /usr/share/nginx/test-domain/  ;\
```

Comment:
you can create multiple directories with one mkdir like:
mkdir -p /usr/share/nginx/test-domain/ /opt/cert